vue/.env

```diff
@@ -0,0 +1,5 @@
+# .env
+VITE_APP_TITLE=PlantMap
+VITE_AUTH_REQUIRED=false
```

Comment:
Couldn't this be removed because the authentication is set in the dev. or prod. env file?
```suggestion
```

Reply:
@jarkenau In principle, this could be removed because they are "overwritten" by `.dev `and `.prod`. I have only left this in as a "fallback" in case you change something in `.dev` or `.prod` or you use a `custom mode` that does not include these variables.